tests/test_meeting.py

```diff
@@ -725,7 +725,7 @@ def test_meeting_get_upcoming_all(client):
         return_data = json.loads(rv.data.decode())
 
         assert return_data["access_token"] == "AccessToken"
-        assert len(return_data["data"]) == 3
+        assert len(return_data["data"]) == 1
```

Comment:
why did this change?

Reply:
I think this was changed to mark that only one meeting is returned from the app

Reply:
Changed what is getting tested